### DISCUSSION TITLE
在msvc下使用 /EHa 选项导致大量刷屏警告，如何解决

### DISCUSSION THREAD

**Discussion by RimuruChan** [xmake-io/xmake#3534] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/42002296/226175427-8c63ec2f-7a21-4dc5-8632-94fa80587f8c.png)

Verbose后发现不知道哪里加入的 /EHsc

![image](https://user-images.githubusercontent.com/42002296/226176792-6c8a32bf-8f31-4a65-b7c4-de36173bcdb6.png)

**Comment by waruqi** [comment, marked as answer]
用 `set_exceptions("none")` 去掉

**Reply by RimuruChan** [reply]
谢谢，希望能更新一下文档，没有查到相关信息，搜源码也没搜到
